nemo/collections/nlp/modules/common/megatron/adapters/adapter_constants.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import enum
+
+
+class AdapterType(str, enum.Enum):
+    # Adapters requires in the IA3 schema
+    KEY_INFUSED = "key_infused_adapter"
+    VALUE_INFUSED = "value_infused_adapter"
+    MLP_INFUSED = "mlp_infused_adapter"
+
+    # Standard Adapters
+    ADAPTER_ONE = "adapter_1"
```

Comment:
Very poor names, please chose something more meaningful.

Reply:
The name change would break backward compatibility. For now, I just want this PR to implement the checking feature. And I'll have another for dealing with backward compatability.